regulations/static/regulations/js/source/views/comment-view.js

```diff
@@ -93,7 +94,15 @@ var CommentView = Backbone.View.extend({
         };
       }).get()
     };
-    window.localStorage.setItem(this.key, JSON.stringify(payload));
+
+    if (payload.comment) {
+      window.localStorage.setItem(this.key, JSON.stringify(payload));
+    }
+    else {
+      // clear comment if input is empty
+      this.clear();
```

Comment:
* What if a user saves a comment with an empty text input but >=1 file attachments? Seems like this would clear their attachments.
* This winds up writing to localStorage twice--first to save the potentially empty comment, then again to remove the key. One write should be enough.
* Will users expect saving an empty comment to remove the comment box from the DOM? That's not what I would expect, but I'm also definitely not our typical user :smile: